contributors/design-proposals/seccomp.md

```diff
@@ -214,6 +248,192 @@ profiles using the key
 `seccomp.security.alpha.kubernetes.io/allowedProfileNames`.  The value of this
 key should be a comma delimited list.
 
+### Spec
+
+We will start from the OCI specification. This API resource will be added to
+`settings.k8s.io` as an `alpha` resource.
+
+```
+// Seccomp represents syscall restrictions
```

Comment:
I would vote for non-namespaced, to match PodSecurityPolicy. It seems like something a ClusterAdmin should have control over... defining seccomp policies and then controlling who has permission to use them.

If an admin of a namespace has permission to create (and then use) new seccomp policies, they could potentially use known vulnerabilities in a multi-tenant cluster, right?